wrapanapi/systems/ec2.py

```diff
@@ -392,10 +392,9 @@ def __init__(self, **kwargs):
             region_name=self._region_name, config=connection_config
         )
 
-        self.elb_connection = ELBConnection(
-            self._username, self._password, region=_regions(
-                regionmodule=elb, regionname=self._region_name)
-        )
+        self.elb_connection = boto3client('elb', aws_access_key_id=self._username,
+            aws_secret_access_key=self._password, region_name=self._region_name,
```

Comment:
Consider an additional enhancement here, since we're re-using the same kwargs when we're calling `boto3client`.  An EC2System._boto3args()` method could provide the kwargs other than the service name itself.

Reply:
Sure, I reworked it differently:
https://github.com/ManageIQ/wrapanapi/pull/362/commits/b52751de618c058a9aac751bb5fa9a563ab06240
This also might be a way?